frontend/src/components/SharedUsersHeader/SharedUsersHeader.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { Typography, Tooltip, IconButton } from '@material-ui/core'
+import { Breadcrumbs } from '../Breadcrumbs'
+import { Icon } from '../Icon'
+import { Title } from '../Title'
+type Props = {
+    onClick?: ()=> void
+  }
+export const SharedUsersHeader:  React.FC<Props> = ({ onClick }) => {
+
+  return (
+    <div>
+      <>
+      <Breadcrumbs />
+      <Typography variant="h1">
+        <Icon name="user-friends" size="lg" />
+        <Title>Shared users</Title>
+        <Tooltip title="Add User">
+          <div>
```

Comment:
I don't think you need the <div> wrapper here unless the tooltip has to support a disabled state.